modules/core/src/v2/baseCoin.ts

```diff
@@ -67,6 +67,8 @@ export interface TransactionParams {
   recipients?: TransactionRecipient[];
   walletPassphrase?: string;
   type?: string;
+  memo?: Memo;
```

Comment:
This is not correct, we can't modify this from base coin just for add Memo and feePayer only for Solana.